x/crosschain/keeper/cctx_gateway_zevm.go

```diff
@@ -23,6 +23,13 @@ func (c CCTXGatewayZEVM) InitiateOutbound(
 	ctx sdk.Context,
 	config InitiateOutboundConfig,
 ) (newCCTXStatus types.CctxStatus, err error) {
+	// abort if CCTX already contains an initial error message from inbound vote msg
+	if config.CCTX.CctxStatus.ErrorMessage != "" {
+		config.CCTX.SetAbort("observation failed", "")
```

Comment:
The second argument should contains that it failed because of depositor fee, and can contains minimum amount and actual amount

Reply:
https://github.com/zeta-chain/node/pull/3358/commits/5e257d5c54ab79d6ead14a71302b84c4aa290315